blocks/top-table-list-block/features/top-table-list/default.jsx

```diff
@@ -84,18 +84,16 @@ class TopTableListWrapper extends Component {
 
 // components end
 export const TopTableList = (props) => {
+	const { customFields = {}, id = "" } = props;
```

Comment:
Any reason for these changes?

Reply:
As mentioned in the huddle, this change was mainly due to the linter flagging these files for review and it coming up.